src/platform/OpenThread/GenericThreadStackManagerImpl_OpenThread.h

```diff
@@ -123,10 +132,18 @@ class GenericThreadStackManagerImpl_OpenThread
         static constexpr uint8_t kMaxInstanceNameSize = chip::Mdns::kMdnsInstanceNameMaxSize;
         static constexpr uint8_t kMaxNameSize         = chip::Mdns::kMdnsTypeAndProtocolMaxSize;
         static constexpr uint8_t kMaxHostNameSize     = 16;
-        // Thread only supports operational discovery
+
+#if CHIP_DEVICE_CONFIG_ENABLE_THREAD_COMMISSIONABLE_DISCOVERY
```

Comment:
Looks like this should more depend on `CHIP_DEVICE_CONFIG_ENABLE_EXTENDED_DISCOVERY`. Discovery is the other part.